soccer/MainWindow.cpp

```diff
@@ -171,8 +175,9 @@ void MainWindow::initialize() {
 
     // Initialize to ui defaults
     on_goalieID_currentIndexChanged(_ui.goalieID->currentIndex());
-    for (const auto& qActionGroup : qActionGroups) {
-        qActionGroup->checkedAction()->trigger();
+    // Do not initialize DefendPlusX(Group 1) or Vision(Group 3)
+    for (int i : {0, 2, 4}) {
```

Comment:
I really don't like these magic numbers.

Is it possible to refer to these by name (maybe `radioGroup`?). Or maybe we could figure out what numbers correspond to these somehow? I'm not sure how QActionGroups work though...